src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -44,12 +63,20 @@ public Drivetrain() {
   @Override
   public void periodic(){
     // TODO 2.2.5: Update odometry
-
-    // TODO 1.2.2: Call tankDrive()
+    tankDrive(Robot.driver.getRawLeftTranslation(), Robot.driver.getRawRightTranslation());
```

Comment:
Use getLeft/RightPosition(), without the "Raw". This squares the joystick results, which makes it easier to control.